jitbuilder/CMakeLists.txt

```diff
@@ -131,6 +131,14 @@ if(OMR_JITBUILDER AND OMR_ENV_DATA64)
 	add_subdirectory(release)
 endif()
 
+if(OMR_JITBUILDER_TEST)
```

Comment:
I just want to verify that your intention is that all three changes described in this commit happen atomically? That's why they're in the same commit?

Reply:
That's correct.